src/targets/browser/index.jsx

```diff
@@ -121,8 +121,8 @@ export const initApp = () => {
   const protocol = window.location ? window.location.protocol : 'https:'
 
   const shareCode = getPublicSharecode()
-  const token = shareCode || data.token
-  const isPublic = Boolean(shareCode || !token || token == '')
+  const token = data.token
```

Comment:
@nono So we can confirm that a token in the body is present in all cases ? I tested successfuly sharing preview write read and read/write access.

Reply:
Yes, I confirm